SegmentEditorLocalThreshold/SegmentEditorLocalThresholdLib/SegmentEditorEffect.py

```diff
@@ -153,6 +153,7 @@ def setMRMLDefaults(self):
     self.scriptedEffect.setParameterDefault(FEATURE_SIZE_MM_PARAMETER_NAME, 3)
     self.scriptedEffect.setParameterDefault(SEGMENTATION_ALGORITHM_PARAMETER_NAME, SEGMENTATION_ALGORITHM_GROWCUT)
     self.scriptedEffect.setParameterDefault(HISTOGRAM_BRUSH_TYPE_PARAMETER_NAME, HISTOGRAM_BRUSH_TYPE_DRAW)
+    self.scriptedEffect.setParameterDefault("TURN_OFF_INTERACTIONS", str(False))
```

Comment:
Always use positive language to make the code easy to read. Instead of turn off = false, it is easier to read interactive=true. Something like `ENABLE_SLICE_VIEW_INTERACTION` would work.